Hobbify-Stats/postgres.py

```diff
@@ -0,0 +1,19 @@
+import psycopg2
+import os
+
+POSTGRES_HOST= os.environ['POSTGRES_HOST']
+POSTGRES_PORT= os.environ['POSTGRES_PORT']
+POSTGRES_DB = os.environ['POSTGRES_DB']
+POSTGRES_USER = os.environ['POSTGRES_USER']
+POSTGRES_PASSWORD = os.environ['POSTGRES_PASSWORD']
+
+def conecct():
+    conn = None
+    if conn == None:
+        try:
+            conn = psycopg2.connect(f'dbname={POSTGRES_DB} user={POSTGRES_USER} password={POSTGRES_PASSWORD} host={POSTGRES_HOST}')
+            return conn
+        except:
+                print('Fail Conecct')
+    else:
+        return conn
```

Comment:
This will never be executed since `conn` starts as `None` by default..

You might also want to look for an exponential backoff strategy when connecting to a DB, that is, try connecting, if it doesn't work, try again after 1.5x the initial time, if it doesn't work, repeat until time reach a limit.